webpack.backend.config.babel.js

```diff
@@ -0,0 +1,34 @@
+import webpack from 'webpack';
```

Comment:
Is this a template as well? If not, could you make some comments explaining what is this file doing?